seed/challenges/08-coding-interview-prep/rosetta-code.json

```diff
@@ -3954,6 +3954,175 @@
+        "testString": "assert(typeof gcd=='function','<code>gcd</code> should be a function.');"
+      }, {
+        "text": "'<code>gcd(24,36)</code> should return a number.'",
+        "testString": "assert(typeof gcd(24,36)=='number','<code>gcd(24,36)</code> should return a number.');"
+      }, {
+        "text": "'<code>gcd(24,36)</code> should return <code>12</code>.'",
+        "testString": "assert.equal(gcd(24,36),12,'<code>gcd(24,36)</code> should return <code>12</code>.');"
+      }, {
+        "text": "'<code>gcd(30,48)</code> should return <code>6</code>.'",
+        "testString": "assert.equal(gcd(30,48),6,'<code>gcd(30,48)</code> should return <code>6</code>.');"
+      }, {
+        "text": "'<code>gcd(10,15)</code> should return <code>5</code>.'",
+        "testString": "assert.equal(gcd(10,15),5,'<code>gcd(10,15)</code> should return <code>5</code>.');"
+      }, {
+        "text": "'<code>gcd(100,25)</code> should return <code>25</code>.'",
+        "testString": "assert.equal(gcd(100,25),25,'<code>gcd(100,25)</code> should return <code>25</code>.');"
+      }, {
+        "text": "'<code>gcd(13,250)</code> should return <code>1</code>.'",
+        "testString": "assert.equal(gcd(13,250),1,'<code>gcd(13,250)</code> should return <code>1</code>.');"
+      }, {
+        "text": "'<code>gcd(1300,250)</code> should return <code>50</code>.'",
+        "testString": "assert.equal(gcd(1300,250),50,'<code>gcd(1300,250)</code> should return <code>50</code>.');"
+      }],
+      "id": "5a23c84252665b21eecc7e82",
+      "challengeType": 5,
+      "releasedOn": "June 7, 2018",
+      "files": {
+        "indexjs": {
+          "key": "indexjs",
+          "ext": "js",
+          "name": "index",
+          "contents": ["function gcd (a,b) {", "  // Good luck!", "}"],
+          "head": [],
+          "tail": []
+        }
+      }
+    },
+    {
+      "title": "Greatest subsequential sum",
+      "description": ["Given a sequence of integers, find a continuous subsequence which maximizes the sum of its elements, that is, the elements of no other single subsequence add up to a value larger than this one.", "An empty subsequence is considered to have the sum of \\( 0 \\);  thus if all elements are negative, the result must be the empty sequence."],
+      "solutions": ["function MaximumSubsequence(population) {\n  function sumValues(arr) {\n      var result = 0;\n      for (var i = 0, len = arr.length; i < len; i++) {\n          result += arr[i];\n      }\n      return result;\n  }\n  var greatest;\n  var maxValue = 0;\n  var subsequence = [];\n\n  for (var i = 0, len = population.length; i < len; i++) {\n      for (var j = i; j <= len; j++) {\n          var subsequence = population.slice(i, j);\n          var value = sumValues(subsequence);\n          if (value > maxValue) {\n              maxValue = value;\n              greatest = subsequence;\n          };\n      }\n  }\n\n  return greatest;\n}\n\n\n"],
```

Comment:
For this challenge, would you mind changing the function name `MaximumSubsequence` to `maximumSubsequence`? Sorry, it's a bit of a pain since they'll have to be changed in the solution, tests, text and text strings, and seed.

I figure this function name should be in camelCase like the rest of them.